pkg/connection/connection.go

```diff
@@ -267,8 +268,9 @@ func (c *csiConnection) Close() error {
 
 func logGRPC(ctx context.Context, method string, req, reply interface{}, cc *grpc.ClientConn, invoker grpc.UnaryInvoker, opts ...grpc.CallOption) error {
 	glog.V(5).Infof("GRPC call: %s", method)
+	glog.V(5).Infof("GRPC request: %s", protosanitizer.StripSecrets(req))
```

Comment:
@xing-yang I noticed snapshotter wasn't logging the request, so I added it back. I also quickly scanned through rest of snapshotter code to see if it was logging requests elsewhere and didn't find anything.  Could you double check?

Reply:
Yes, we removed the logging of the request when Jan reviewed it due to the secrets issue.  This is the only spot.